apps/wallet/src/ui/app/pages/home/onramp/index.tsx

```diff
@@ -32,7 +31,15 @@ export function Onramp() {
         if (providers && providers.length === 0) {
             navigate('/tokens');
         }
-    }, [providers, navigate]);
+    }, [providers, navigate, mutate]);
+
+    useEffect(() => {
```

Comment:
I think this should be a query instead of a mutation since we're only calling in declaratively on render

Reply:
yea, I agree - this was Jordan's code so wasn't sure what the intentions were behind making it a mutation